CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## v0.20.0 - 2024-10-17
+
+### 🚀 Enhancements
+- Add 1.31 support and drop 1.26 @zeitlerc [#574](https://github.com/newrelic/k8s-metadata-injection/pull/574)
+
```

Comment:
```suggestion

### enhancement 
- Add 1.31 support and drop 1.26 @zeitlerc [#574](https://github.com/newrelic/k8s-metadata-injection/pull/574) 
```

the fun changelog w/ the emojis and stuff are generated. remove lines 10-13 and replace w/ the suggested change. sorry changelog sucks

Reply:
https://github.com/newrelic/release-toolkit/blob/main/README.md#validate-markdown